exe-common/src/config.rs

```diff
@@ -49,13 +49,16 @@ pub mod net {
     pub enum Peer {
         Native(String),
         Http(String),
+        Ntt(String),
     }
     impl Peer {
         /// analyse the content of the given `addr` and construct the correct kind
         /// of `Peer` accordingly.
         pub fn new(addr: String) -> Self {
             if addr.starts_with(r"http://") || addr.starts_with(r"https://") {
                 Peer::http(addr)
+            } else if addr.starts_with(r"x-cardano-ntt://") {
+                Peer::ntt(addr["x-cardano-ntt://".len()..].to_string())
```

Comment:
OK, though using a string constant would be better.

Reply:
It's also not an `r`-literal, though I'm not sure there is a practical difference. Small things...